proto/tendermint/abci/types.proto

```diff
@@ -167,9 +167,11 @@ message ResponseInfo {
   string version     = 2;
   uint64 app_version = 3;
 
-  int64  last_block_height             = 4;
-  bytes  last_block_app_hash           = 5;
-  uint32 last_core_chain_locked_height = 100;
+  int64 last_block_height   = 4;
+  bytes last_block_app_hash = 5;
+ 
+  reserved "last_core_chain_locked_height";
+  reserved 100;
```

Comment:
Do we keep it for backward compatibility?

Reply:
Yes, it's reserved for backward compatibility, so that when we add another field, we will not reuse 100 nor the label.

Reply:
What if just remove it since we update Drive as well?

Reply:
As far as I understand the idea of Protobuf, it should stay there forever (until major release, if we follow Semantic Versioning), in order to ensure full backward compatibility.

Reply:
Sounds good to me